src/Graphics/Drawing.purs

```diff
@@ -245,6 +240,46 @@ rotate = Rotate
 text :: Font -> Number -> Number -> FillStyle -> String -> Drawing
 text = Text
 
+-- | Get the size of a Drawing.
+drawingSize :: Drawing -> { w:: Number, h:: Number }
+drawingSize (Fill shape _) = shapeSize shape
+drawingSize (Outline shape (OutlineStyle oStyle)) = shapeSize shape -- TODO
+drawingSize (Text font x y fillStyle string) = { w: 0.0, h: 0.0} -- TODO
```

Comment:
I wonder how one would go about doing this? I seems you'd need to access [Canvas' measureText](https://github.com/purescript-contrib/purescript-canvas/blob/master/docs/Graphics/Canvas.md#measuretext)?